wishlist.toml

```
@@ -89,6 +89,13 @@ upstream = "https://github.com/anyproto/any-sync"
 website = "https://anytype.io"
 added_date = 1720804319  # 2024/07/12
 
+
+[appertium]
+name = "Appertium"
+description = "A machine translation platform"
+upstream = "https://github.com/apertium"
```

```suggestion
upstream = "https://github.com/apertium/apertium-html-tools"
```

After spending way too much time understanding the architecture of this project, this repo should be the one to be packaged, but it depends on the [API server](https://wiki.apertium.org/wiki/Apertium-apy) and the [apertium package](https://wiki.apertium.org/wiki/Install_Apertium_core_using_packaging) too